src/generate.rs

```diff
@@ -367,11 +406,27 @@ mod tests {
         assert!(errors.is_empty());
     }
 
+    #[test]
+    fn generate_configfile() {
+        let filepath = PathBuf::from("./config.example.toml");
```

Comment:
You should do this in a tempdir so that you don't dirty the users repo on a failure.

Reply:
Oh! Yep, good catch!